app/i18n/locales/en/translation.json

```diff
@@ -81,15 +98,15 @@
   "expenses_summary_title": "Expenses Summary",
   "expenses_summary_total": "Total expenses added: ${{amount}}",
   "expenses_summary_yes_delete_item": "Yes, remove this expense",
-  "how_this_works_header": "How this works",
-  "how_this_works_accordion_what_is_self_employment_title": "What is self-employed work?",
   "how_this_works_accordion_what_expenses_count_title": "What expenses count?",
+  "how_this_works_accordion_what_is_self_employment_title": "What is self-employed work?",
```

Comment:
This `self-employed` has a hyphen, but the others do not. Need to ask the design team if there should be a hyphen or not.